.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+## Related issue
+- One line for each issue. If merging this pull request should auto-close the issue on that line,
+preface the issue URL/number with "Fixes". Every pull request should have at least one issue
+linked.
+
+## Proposed changes
```

Comment:
You can remove this section now. It duplicates the "Summarizie your changes" section below.